content/en-us/reference/engine/classes/AudioLimiter.yaml

```
@@ -29,8 +33,10 @@ properties:
       - Audio
     writeCapabilities: []
   - name: AudioLimiter.MaxLevel
-    summary: ''
-    description: ''
+    summary: 'The maximum volume tolerated.'
+    description: |
+      The maximum volume, in decibels, that the limiter will allow to pass through, unclamped.
+      Whenever 
```

It looks like you meant to write something here?

---

You're right lol; probably
```suggestion
      Whenever the input stream exceeds `Class.AudioLimiter.MaxLevel`, the output stream's volume will be reduced to compensate.
```